scripts/zones/Caedarva_Mire/npcs/Nareema.lua

```diff
@@ -21,25 +21,30 @@ entity.onTrigger = function(player, npc)
     local beginnings = player:getQuestStatus(tpz.quest.log_id.AHT_URHGAN, tpz.quest.id.ahtUrhgan.BEGINNINGS)
 
     -- IMMORTAL SENTRIES
-    if (toauMission == tpz.mission.id.toau.IMMORTAL_SENTRIES) then
-        if (player:hasKeyItem(tpz.ki.SUPPLIES_PACKAGE)) then
+    if toauMission == tpz.mission.id.toau.IMMORTAL_SENTRIES then
```

Comment:
These npcs for the mission and assault armband handling g
are completely handled through the assault global